qa/integration/src/test/resources/features/authorization/GroupService.feature

```diff
@@ -256,3 +256,1053 @@ Feature: Group Service tests
+    And I search for a group named "Group1"
+    Then Device is in groups Assigned Devices
+    When I search for a device with the client ID "Device2"
+    And I search for a group named "Group1"
+    Then Device is in groups Assigned Devices
+    And No exception was thrown
+    And I logout
+
+  Scenario: Assign 100 Devices to One Group
+  Login as kapua-sys, go to groups, create a group "Group1".
+  Go to devices, create 100 devices and add them to created group.
+  Check if created devices are in Assigned Devices of the group.
+  Kapua should not return any errors.
+
+    Given I login as user with name "kapua-sys" and password "kapua-password"
+    And I configure the group service
+      | type    | name                   | value | scopeId | parentId |
+      | boolean | infiniteChildEntities  | true  | 0       | 1        |
+      | integer | maxNumberChildEntities | 5     | 0       | 1        |
+    And I configure the device registry service
+      | type    | name                   | value |
+      | boolean | infiniteChildEntities  | true  |
+      | integer | maxNumberChildEntities | 10    |
+    When I create the group with name "Group1"
+    And I create 100 devices and add them to group
+    And I search for a group named "Group1"
+    And I search for a device with the client ID "Device00"
+    Then Device is in groups Assigned Devices
+    When I search for a device with the client ID "Device55"
+    Then Device is in groups Assigned Devices
+    When I search for a device with the client ID "Device99"
+    Then Device is in groups Assigned Devices
+    Then No exception was thrown
+    And I logout
+
+  Scenario: Adding Regular Group With Description to Device
+  Login as kapua-sys, go to groups, create a group with description.
+  Go to devices, create a device and add created group to it.
+  Check if created device is in Assigned Devices of the group.
+  Kapua should not throw any errors
+
+    Given I login as user with name "kapua-sys" and password "kapua-password"
+    And I configure the group service
+      | type    | name                   | value | scopeId | parentId |
+      | boolean | infiniteChildEntities  | true  | 0       | 1        |
+      | integer | maxNumberChildEntities | 5     | 0       | 1        |
+    And I configure the device registry service
+      | type    | name                   | value |
+      | boolean | infiniteChildEntities  | true  |
+      | integer | maxNumberChildEntities | 10    |
+    When I create the group with name "Group1" and description "Description 1234"
+    And I create a device with name "Device1"
+    And I add device to group
+    And I search for a group named "Group1"
+    Then Device is in groups Assigned Devices
+    And No exception was thrown
+    And I logout
+
+  Scenario: Change Group With Description of Device
+  Login as kapua-sys, go to groups, create a group.
+  Go to devices, create a device ("Device1") and add created group to it.
```

Comment:
Change description to: "Go to devices, create a device ("Device1") and add it to the created group."